src/app.js

```diff
@@ -228,9 +228,16 @@ Api.prototype.registerRoute = function (cb) {
 Api.prototype.connectDb = function () {
 
 
-    var dbUri = 'mongodb://' + this.config.get('db.mongo.host') + '/' + this.config.get('db.mongo.name');
+    var configDbMonggo = this.config.get('db.mongo');
+    var dbUri = '';
 
-    this.mongo.connect(dbUri);
+    if(_.isObject(configDbMonggo)){
+        dbUri = 'mongodb://' + this.config.get('db.mongo.host') + '/' + this.config.get('db.mongo.name');
+    } else if(_.isString(configDbMonggo)) {
+        dbUri = 'mongodb://' + configDbMonggo;
```

Comment:
I would recommend that the config setting include the protocol identifier `mongodb://` instead of appending it here manually. The config string should be a full, valid mongo URI. That will make the configuration setting more clear and more specific.